tracks/latest.toml

```diff
@@ -61,4 +61,5 @@ full = [
     "../specs/union.toml",
     "../specs/ignore3vl.toml",
     "../specs/lookup_join.toml",
+    "..specs/select/topk_aggregations.toml",
```

Comment:
Does this path variant work? all other have a slash after the `..`

Reply:
👍